bundler/helpers/v2/monkey_patches/bundler_spec_set_patch.rb

```diff
@@ -0,0 +1,23 @@
+# typed: false
+# frozen_string_literal: true
+
+require "bundler/definition"
```

Comment:
```suggestion
require "bundler/spec_set"
```

Reply:
Maybe we need to require `bundler/lazy_specification` and whatever defines `Bundler::GemNotFound` also